benefits/core/models.py

```diff
@@ -167,24 +167,24 @@ class EligibilityVerifier(models.Model):
     name = models.TextField()
     display_order = models.PositiveSmallIntegerField(default=0, blank=False, null=False)
     active = models.BooleanField(default=False)
-    api_url = models.TextField(null=True, blank=True)
-    api_auth_header = models.TextField(null=True, blank=True)
-    api_auth_key_secret_name = SecretNameField(null=True, blank=True)
+    eligibility_api_url = models.TextField(null=True, blank=True)
+    eligibility_api_auth_header = models.TextField(null=True, blank=True)
+    eligibility_api_auth_key_secret_name = SecretNameField(null=True, blank=True)
     eligibility_type = models.ForeignKey(EligibilityType, on_delete=models.PROTECT)
     # public key is used to encrypt requests targeted at this Verifier and to verify signed responses from this verifier
-    public_key = models.ForeignKey(PemData, related_name="+", on_delete=models.PROTECT, null=True, blank=True)
+    eligibility_api_public_key = models.ForeignKey(PemData, related_name="+", on_delete=models.PROTECT, null=True, blank=True)
     # The JWE-compatible Content Encryption Key (CEK) key-length and mode
-    jwe_cek_enc = models.TextField(null=True, blank=True)
+    eligibility_api_jwe_cek_enc = models.TextField(null=True, blank=True)
     # The JWE-compatible encryption algorithm
-    jwe_encryption_alg = models.TextField(null=True, blank=True)
+    eligibility_api_jwe_encryption_alg = models.TextField(null=True, blank=True)
     # The JWS-compatible signing algorithm
-    jws_signing_alg = models.TextField(null=True, blank=True)
+    eligibility_api_jws_signing_alg = models.TextField(null=True, blank=True)
     auth_provider = models.ForeignKey(AuthProvider, on_delete=models.PROTECT, null=True, blank=True)
     selection_label_template = models.TextField()
-    start_template = models.TextField(null=True, blank=True)
+    eligibility_start_template = models.TextField(null=True, blank=True, default="eligibility/start.html")
```

Comment:
Since a default value is now provided, I think this field shouldn't allow `null=True, blank=True` because every `EligibilityVerifier` should have a value. Previously it could be null/blank because if so, the code would provide a value.

Reply:
Yes, great point.

Reply:
Updated in commit [`15ef73c`](https://github.com/cal-itp/benefits/pull/2272/commits/15ef73cfbf2eca723e3c902e893c31372338274a)